README.md

```diff
@@ -17,6 +17,16 @@ pip install ytcc
 Install [ytcc-git](https://aur.archlinux.org/packages/ytcc-git/) from the AUR.
 The [ytcc](https://aur.archlinux.org/packages/ytcc/) package will be upgraded to version 2.0.0, when it has a stable release.
 
+### Gentoo
```

Comment:
@EmRowlands Could you please add a note that this is maintained by you?
I want that note, because it informs users that the Gentoo ebuild is maintained by someone else than the ytcc developer and gives you the credit for packaging it.

```suggestion
### Gentoo (maintained by [@EmRowlands](https://github.com/EmRowlands))
```

Reply:
Absolutely! I chose slightly different formatting, but the effect is the same.